internal/api/v1/application/deploy.go

```diff
@@ -260,3 +272,29 @@ func newAppService(app models.AppRef, username string) *v1.Service {
 		},
 	}
 }
+
+// replaceInternalRegistry replaces the registry part of ImageURL with the localhost
```

Comment:
How similar is this function to function `registry.ReplaceWithInternalRegistry` ? Could it be replaced by it ?

Reply:
> calls the other

:facepalm: missed that. No objections to any changes.